packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -457,7 +462,7 @@ export class GumbyController {
 
     private async handleError(message: { error: Error; tabID: string }) {
         if (message.error instanceof AlternateDependencyVersionsNotFoundError) {
-            this.messenger.sendKnownErrorResponse('no-alternate-dependencies-found', message.tabID)
+            this.messenger.sendKnownErrorResponse(message.tabID, CodeWhispererConstants.dependencyVersionsErrorMessage)
```

Comment:
Out of curiosity why is this dependency version error message stored in codewhisperer constants when it has to do with gumby code?

Reply:
All of our constants are in CodeWhisperer constants because that's how QCT was originally intended. Now that we're entirely separate, a future PR is needed to move those constants.